docs/commands/command-line-interface.md

```diff
@@ -131,6 +134,7 @@ Clear unchecked blocks
 
 ### --vacuum
 Compact database. If data_path is missing, the database in data directory is compacted. Optional `--unchecked_clear`, `--clear_send_ids`, `--online_weight_clear`, `--peer_clear`
+Optional `--rebuild_database` in version 21.0+
```

Comment:
Do we need to call out the *2 disk space here as well?

Reply:
Should this be mentioned on `--snapshot` section as well? Also `--confirmation_height_clear` should probably be mentioned in vacuum too.

Reply:
Why mention this in `--snapshot`?